src/Integration/MefServices/VsSessionHost.cs

```diff
@@ -269,36 +260,8 @@ private BindingConfiguration SafeGetBindingConfig()
 
         #region IServiceProvider
 
-        private void RegisterLocalServices()
-        {
-            var projectNameTestProjectIndicator = new Lazy<ILocalService>(() => new ProjectNameTestProjectIndicator(Logger));
-            localServices.Add(typeof(ITestProjectRegexSetter), projectNameTestProjectIndicator);
-
-            Debug.Assert(SupportedLocalServices.Length == localServices.Count, "Unexpected number of local services");
-            Debug.Assert(SupportedLocalServices.All(t => localServices.ContainsKey(t)), "Not all the LocalServices are registered");
-        }
-
-        public object GetService(Type serviceType)
-        {
-            // We don't expect COM types, otherwise the dictionary would have to use a custom comparer
-            Lazy<ILocalService> instanceFactory;
-            if (typeof(ILocalService).IsAssignableFrom(serviceType) &&
-                this.localServices.TryGetValue(serviceType, out instanceFactory))
-            {
-                return instanceFactory.Value;
-            }
+        public object GetService(Type serviceType) => serviceProvider.GetService(serviceType);
```

Comment:
Taking out all of the special-case code for local services just leaves this as a pass-through.

Logically, `IHost` doesn't need to be an `IServiceProvider` anymore since it isn't providing any services. However, I suspect that would create a lot of churn.